packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -84,6 +85,13 @@ export default class QuanticGeneratedAnswer extends LightningElement {
    * @default {false}
    */
   @api multilineFooter;
+  /**
+   * Indicates whether the disclaimer should be hidden.
+   * @api
+   * @type {boolean}
+   * @default {false}
+   */
+  @api noDisclaimer;
```

Comment:
Just my opinion but I feel that naming the property `withoutDisclaimer` or `hideDisclaimer` feels a bit more natural.

Plus if we keep the public property it will need to be tested with Cypress as well.

Reply:
Also is really necessary to have this property, like we are already exposing a slot for the disclaimer so we already have a way to get a generated answer component without a disclaimer.

I assume that the majority of the people using this component would want the disclaimer since it's useful and they can customize its content.
for the few/rare users who don't want a disclaimer, they can remove it but specifying an empty slot, I know it would not be a super fancy way  to hide a disclaimer, but considering that only very few people will need to do that I would not add a new public property just for this particular situation. 

The idea is I don't see the point of having a public property that allows you to do something that you can achieve without using this property.

Reply:
Do you know if we can empty a slot? If we can then yes, I wouldn't have this property

Reply:
This should do it:
```
   <c-quantic-generated-answer
      engine-id={engineId}
      answer-style={answerStyle}
      multiline-footer={multilineFooter}
    >
      <div slot="disclaimer"></div>
   </c-quantic-generated-answer>
```

Reply:
ok then I'm happy to remove the option :)